extract.py

```diff
@@ -162,7 +162,7 @@ def _ignore_text(content, style):
 
     rgb_color = get_dict_nested_value(style, 'foregroundColor', 'opaqueColor', 'rgbColor')
 
-    if not rgb_color or len(rgb_color) == 0:
+    if not rgb_color or len(rgb_color) == 0 or rgb_color == {'red': 0.13333334, 'blue': 0.13333334, 'green': 0.13333334}:
```

Comment:
feels like the slides are using the wrong colour rather than the code being wrong, I'd update the slides and remove this (e.g. select the offending text and change it to black)